app/src/online/java/com/kamron/pogoiv/CrashlyticsWrapper.java

```diff
@@ -30,10 +26,16 @@ public static void init(Context context) {
 
     public static class CrashReportingTree extends Timber.Tree {
 
+        public final Context context;
+
+        public CrashReportingTree(Context context) {
+            this.context = context;
+        }
+
         @Override
         protected void log(int priority, String tag, String message, Throwable t) {
 
-            if (BuildConfig.isInternetAvailable && GoIVSettings.getSettings(mContext).getSendCrashReports()) {
+            if (BuildConfig.isInternetAvailable && GoIVSettings.getSettings(context).getSendCrashReports()) {
```

Comment:
Nah building settings each time seems unnecessary. It looks like restarting the application is required to pick up that change in settings. There's maybe a thornier problem in that there's no obvious way to turn crash reporting off after Fabric has been initialised.

That's a problem for another PR though